framework/include/tinyalsa/tinyalsa.h

```diff
@@ -398,6 +398,75 @@ unsigned int pcm_format_to_bits(enum pcm_format format);
  */
 unsigned int pcm_get_subdevice(const struct pcm *pcm);
 
+/**
+* @brief Writes audio samples to the PCM using the mmap buffer
+*
+* @details @b #include <tinyalsa/tinyalsa.h>
+* @param[in] pcm A PCM handle
+* @param[in] data Pointer to the buffer containing data to be written
+* @param[in] count Size of the data in bytes
+* @returns On success, written number of frames returned. On failure, a negative number returned
+* @since Tizen RT v1.x
+*/
+int pcm_mmap_write(struct pcm *pcm, const void *data, unsigned int count);
```

Comment:
utc should test api what you added here.

Reply:
Yes. I understand. But, this mmap_write API is actually not useful. Internally it performs memcpy which we want to avoid. So, either we can remove this API, or we can just keep it and write an additional UTC for it. 

However, i suggest that this api must not be used by application. Application must use the other mmap api's like avail -> wait -> begin -> commit explicitly to take advantage of mmap.

Reply:
Added testcases for all APIs.
Still, Application must use the other mmap api's like avail -> wait -> begin -> commit explicitly to take advantage of mmap.